test/unit/Reflection/ReflectionFunctionAbstractTest.php

```diff
@@ -133,6 +136,30 @@ public function testIsDeprecated() : void
         self::assertFalse($function->isDeprecated());
     }
 
+    public function deprecatedStubProvider(): array
+    {
+        return [
+            ['create_function', 70400, true],
+            ['create_function', 70100, false],
+            ['zip_open', 70400, false],
+            ['zip_open', 80000, true],
+        ];
+    }
+
+    /**
+     * @dataProvider deprecatedStubProvider
+     */
+    public function testStubbedIsDeprecated(string $functionName, int $phpVersionId, bool $expectedDeprecated): void
+    {
+        BetterReflection::$phpVersion = $phpVersionId;
```

Comment:
I think some of the new failures are caused by the fact that you're overwriting this property. You can simply pass $phpVersionId into the PhpStormStubsSourceStubber constructor instead.

Compare the failures between your PR: https://github.com/ondrejmirtes/BetterReflection/pull/11/checks?check_run_id=3832468090 (Tests: 8773, Assertions: 86564, Errors: 164, Failures: 138, Warnings: 1, Skipped: 8.)

And master: https://github.com/ondrejmirtes/BetterReflection/runs/3802415862?check_suite_focus=true (Tests: 8769, Assertions: 121172, Failures: 138, Warnings: 1, Skipped: 8.)

Reply:
Oh! I thought I copied/borrowed that from somewhere. Ah, `ReflectionClassStringableTest`.

Without overriding that value, the tests provided by the PR fail inside of the ReflectionFunctionAbstract:

```
           if ($deprecatedTag) {
                $sinceId = PhpIdParser::fromVersion($deprecatedTagMatches[1]);
                return BetterReflection::$phpVersion >= $sinceId;
            }
                if (isset($attributeArgs['since'])) {
                    $sinceId = PhpIdParser::fromVersion($attributeArgs['since']);
                    return BetterReflection::$phpVersion >= $sinceId;
                }
```

Only because we try to test old PHP version asserts.

Reply:
I did a commit & push to add that same tearDown and will see what happens (..or not, because GitHub won't run my workflows yet :( )